search/includes/classes/class-search.php

```diff
@@ -614,24 +628,24 @@ public static function record_ratelimited_query_stat() {
 		$statsd->increment( $per_site_stat );
 	}
 
-	public function set_queue_wait_time_gauge() {
-		$indexable = \ElasticPress\Indexables::factory()->get( 'post' );
+	public function maybe_alert_for_average_queue_time() {
+		$indexable = $this->indexables->get( 'post' );
 
 		if ( ! $indexable ) {
 			return;
 		}
 
 		$average_wait_time = $this->queue->get_average_queue_wait_time();
 
-		$statsd_mode = 'queue_wait_time';
-		$statsd_index_name = $indexable->get_index_name();
-
-		$url = $this->get_current_host();
-		$per_site_stat = $this->get_statsd_prefix( $url, $statsd_mode, FILES_CLIENT_SITE_ID, $statsd_index_name );
-
-		$statsd = new \Automattic\VIP\StatsD();
-
-		$statsd->gauge( $per_site_stat, $average_wait_time );
+		if ( $average_wait_time > self::STALE_QUEUE_WAIT_LIMIT ) {
+			$message = sprintf(
+				'Average index queue time for application %d, exceeded %d seconds with %d seconds',
```

Comment:
```suggestion
				'Average index queue wait time for application %d is currently %d seconds',
```

I would suggest putting the current queue wait time before the threshold, for ease of reading. We can also just leave the threshold off entirely here. 

We should add the site's domain though, like we do on other alerts.

Reply:
Fixed